src/stories/target.stories.js

```diff
@@ -0,0 +1,51 @@
+import { buildWikipediaUrl } from '../utils'
+import { init } from '../index'
+
+export default {
+	title: 'Target Cases',
+	argTypes: {
+		lang: {
+			name: 'Language',
+			defaultValue: 'en',
+			control: {
+				type: 'select',
+				options: [ 'en', 'fr', 'hi', 'ks', 'he' ]
+			}
+		},
+		title: {
+			name: 'Article Title',
+			defaultValue: 'Cat',
+			control: 'text'
+		}
+	}
+}
+
+export const Hyperlink = ( { lang, title } ) => {
+	const container = document.createElement( 'div' )
+	const template = `<a href="${buildWikipediaUrl( lang, title, true, false )}">${title} (${lang})</a>`
+	container.innerHTML = template
+	container.onmouseover = () => {
```

Comment:
Have you tried calling the init function directly instead of putting it in the mouseover handler? The `root` option should be the `container` variable.